lib/routes/tencent/cloud/developer/column.ts

```diff
@@ -62,17 +63,19 @@ export const route: Route = {
 };
 
 async function findClassifyById(id) {
-    const classifylink = `https://cloud.tencent.com/developer/api/column/get-classify-list-by-scene`;
-    const response = await ofetch(classifylink, {
-        method: 'POST',
-        headers: {
-            'Content-Type': 'application/json',
-        },
-        body: {
-            scene: 0,
-        },
+    const classifylink = 'https://cloud.tencent.com/developer/api/column/get-classify-list-by-scene';
+    const result = await cache.tryGet(classifylink, async () => {
+        const response = await ofetch(classifylink, {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+            },
+            body: {
+                scene: 0,
+            },
+        });
+        return response.list.find((classify) => classify.id === Number(id));
```

Comment:
> Please cache the `response`.

Cache the HTML `response` only. Otherwise, `findClassifyById` will return the same thing for different `id`.

Reply:
done